bind/config.sls

```diff
@@ -169,7 +169,7 @@ bind_rndc_client_config:
 {% endif %}
 
 {% for zone, zone_data in salt['pillar.get']('bind:configured_zones', {}).items() -%}
-{%- set file = salt['pillar.get']("bind:available_zones:" + zone + ":file", zone_data.get('file')) %}
```

Comment:
This change was introduced [here](https://github.com/saltstack-formulas/bind-formula/pull/71/commits/2156b92221186be32ba1324a96acb1077fccdc8d) on PR #71, but seems unrelated to the scope of that PR.

Reply:
Indeed.